doc/ref_arch/kubernetes/chapters/chapter02.rst

```diff
@@ -4,8 +4,8 @@ Architecture Requirements
 Introduction
 ------------
 
-This chapter will use the requirements defined in the overall Reference Model and only make additional entries in section
-`2.3 <#2.3>`__ if there are additional requirements needed for this Reference Architecture.
+This chapter will use the requirements defined in the overall Reference Model and only make additional entries in
+section `2.3 <#2.3>`__ if there are additional requirements needed for this Reference Architecture.
```

Comment:
can we not use section numbers as they may change?

section `Reference Model Requirements`__ if there are additional ...